### PR TITLE
Add example for roll printing

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: site/en/docs/extensions/reference/printing/index.md [hunk 1]
```diff
@@ -59,17 +59,17 @@ const defaultPrinter = printers.find((printer) => {
...
 ```
 
-### submitJob()
+### submitJob() {: #submitjob }
 
 The `submitJob()` method requires three things.
 
 * A `ticket` structure specifying which capabilities of the printer are to be used. If the user needs to select from available capabilities, you can retrieve them for a specific printer using `getPrinterInfo()`.
-* A `SubmitJobRequest` structure, which specifies the printer to use, and the file or date to print. This structure contains a reference to the `ticket` structure. 
+* A `SubmitJobRequest` structure, which specifies the printer to use, and the file or date to print. This structure contains a reference to the `ticket` structure.
```
Comment (jpmedley) [inline site/en/docs/extensions/reference/printing/index.md:67]: I turned on an option in VSCode that trims whitespace differences. Sorry about that.

File: site/en/docs/extensions/reference/printing/index.md [hunk 2]
```diff
@@ -97,4 +97,60 @@ chrome.printing.submitJob(submitJobRequest, (response) => {
...
+    "display_name": "finishings/11",
+    "id": "finishings/11",
+    "type": "TYPED_VALUE",
+    "typed_value_cap": {
+      "value_type": "BOOLEAN"
+    }
+  },
+  ...
+]
+      ```
```
Comment (AmySteam) [inline site/en/docs/extensions/reference/printing/index.md:139]:
```suggestion
```
```

File: site/en/docs/extensions/reference/printing/index.md [hunk 3]
```diff
@@ -97,4 +97,60 @@ chrome.printing.submitJob(submitJobRequest, (response) => {
...
+
+This example shows how to build a printer ticket for continuous (or roll) printing, which is often used with receipt printing. The `submitJobRequest` object for roll printing is the same as that shown for the [`submitJob()`](#submitjob) example.
+
+If you need to change the default value for paper cutting, use the `vendor_ticket_item` key. (The default varies from printer to printer.) When included, this key needs to be an array with one member: an object whose `id` is `'finishings'`. The value can either be `'trim'` for printers that cut the roll at the end of printing or `'none'` for printers that require the print job to be torn off.
+
+```json/3
+var ticket = {
+  version: '1.0',
+  print: {
+    vendor_ticket_item = [{id: 'finishings', value: 'trim'}],
```
Comment (AmySteam) [inline site/en/docs/extensions/reference/printing/index.md:110]:
```suggestion
const ticket = {
  version: '1.0',
  print: {
    vendor_ticket_item: [{id: 'finishings', value: 'trim'}],
```

File: site/en/docs/extensions/reference/printing/index.md [hunk 4]
```diff
@@ -97,4 +97,60 @@ chrome.printing.submitJob(submitJobRequest, (response) => {
...
+};
+```
 
+Some printers do not support the `"finishings"` option. To determine if your printer does, call [`getPrinterInfo()`](#method-getPrinterInfo) and look for a `"display_name"` of `"finishings/11"`.
+
+```json/2
+"vendor_capability": [
+  {
+    "display_name": "finishings/11",
+    "id": "finishings/11",
```
Comment (AmySteam) [inline site/en/docs/extensions/reference/printing/index.md:131]: Maybe the id can be different from the display name?

File: site/en/docs/extensions/reference/printing/index.md [hunk 5]
```diff
@@ -97,4 +97,60 @@ chrome.printing.submitJob(submitJobRequest, (response) => {
...
+"media_size": {
+  "option": [
+  {
+    "custom_display_name": "",
+    "is_continuous_feed": true,
+    "max_height_microns": 2000000,
+    "min_height_microns": 25400,
+    "width_microns": 50800
+  },
+  …
```
Comment (AmySteam) [inline site/en/docs/extensions/reference/printing/index.md:153]:
```suggestion
  ...
```

**Review by AmySteam — CHANGES_REQUESTED** [review]
Left a couple of comments on the code samples, but the content is spot on 👍 💯

File: site/en/docs/extensions/reference/printing/index.md (hunk 4 above)
Comment (jpmedley) [reply site/en/docs/extensions/reference/printing/index.md:131]: These are vendor defined. So we can't change them.

File: site/en/docs/extensions/reference/printing/index.md [hunk 6]
```diff
@@ -97,4 +97,59 @@ chrome.printing.submitJob(submitJobRequest, (response) => {
...
+    "display_name": "finishings/11",
+    "id": "finishings/11",
+    "type": "TYPED_VALUE",
+    "typed_value_cap": {
+      "value_type": "BOOLEAN"
+    }
+  },
+  ...
+]
+
```
Comment (AmySteam) [inline site/en/docs/extensions/reference/printing/index.md:139]:
```suggestion
```

```

**Pull request merged**